components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -14,19 +14,33 @@
 
 #import <UIKit/UIKit.h>
 
-@class MDCButton;
+#import "MaterialButtons.h"
 
 __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
 
-@property(nonatomic, copy, nonnull) NSString *text;
```

Comment:
I notice it's nonnull. Is the default just an empty string?

Reply:
It was an empty string. It has been changed to nonnull UILabel `textLabel` now, so its string `textLabel.text` will be nullable.